plugin-server/src/capabilities.ts

```diff
@@ -106,5 +106,10 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
                 cdpFunctionOverflow: true,
                 ...sharedCapabilities,
             }
+        case PluginServerMode.sync_inline_plugins:
+            return {
+                syncInlinePlugins: true,
```

Comment:
I'm not sure if we'd want to add a separate `PluginServerMode` for it, but instead add the capability to scheduler like appManagementSingleton? We won't be running a separate plugin-server for just this.
Also add it to null `PluginServerMode` too, i.e. local testing and hobby deploy.

Reply:
I was figuring the plugin server would be run once in "sync_inline_plugins" mode, as deployment job, rather than having syncing being part of _any_ regular server deployment, but i've moved it to be a scheduler capability, I think that's fine. Also added to the null one.

Reply:
oh I see, that could work too and would potentially be better ... we kinda want the inline plugins to be synced before the deploy and if that fails it should block the deployment